packages/web-components/fast-components-msft/src/design-system-provider/index.ts

```diff
@@ -0,0 +1,7 @@
+import {
+    designSystemProvider,
+    FASTDesignSystemProvider,
+} from "@microsoft/fast-components";
+
+@designSystemProvider("msft-design-system-provider")
+export class MSFTDesignSystemProvider extends FASTDesignSystemProvider {}
```

Comment:
@nicholasrice does this map to what you might expect here? I'd expect the default values to change here once we create the FAST system.

Reply:
Yep